oceannavigator/frontend/src/components/ComboBox.jsx

```
@@ -269,58 +265,60 @@ class ComboBox extends React.Component {
-        helpOptions = this.state.data.map(function(d) {
+      if (
+        this.state.data.length > 1 &&
+        this.state.data.slice(-1)[0].hasOwnProperty("help")
+      ) {
+        helpOptions = this.state.data.map(function (d) {
           return (
-            <p key={d.id}><em>{d.value}</em>:
-              <span dangerouslySetInnerHTML={{ __html: d.help}} />
+            <p key={d.id}>
+              <em>{d.value}</em>:
+              <span dangerouslySetInnerHTML={{ __html: d.help }} />
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)